### ISSUE TITLE
Stop asking for support here.

### ISSUE THREAD

**Issue by kpwn** [kpwn/yalu102#64]
title

**Comment by lolman8776** [comment]
All of this. This this this.

**Comment by kmddd59** [comment]
So your saying don't report issues ok gotcha

**Comment by zd9** [comment]
No, he is saying don't do stuff like #76 #77 or any of the other issues that say "{my device type} support now plz"

**Comment by GusToughSon** [comment]
If You = Not Able To Comprehend Then
Move on 
Else
If your contributing to the code Then
@Error then
make issue
endif
endif

**Comment by evildog1** [comment]
This is not the only one that got useless issue reports.
Peoples who modding/hacks games are also having spam reports and begging for hacks
